Source/NaturalLanguageClassifierV1/Models/Classification.swift

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright IBM Corporation 2015
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import Foundation
+import Freddy
+
+extension NaturalLanguageClassifierV1 {
+    
+    public struct Classification: JSONDecodable {
+        
+        /// Unique identifier for the classifier used.
+        public var classifierId: String
```

Comment:
Would you mind switching all `var` properties in the `NaturalLanguageClassifierV1` models to `let` instead?

Previously, ObjectMapper required all properties to be optional and mutable--it worked by creating an empty model and then going through and populating each property. One of the advantages of Freddy, though, is that we decode and set properties in the `init`. So unlike ObjectMapper, Freddy doesn't require us to have `var` properties.

Since Freddy gives us more freedom, it raises a question: should the properties be mutable (`var`) or immutable (`let`)? I think our models should use `let`, since we never expect these properties to change. This also expresses our expectations of immutability to the SDK's consumer.

On a related note, ObjectMapper previously required all properties to be optional. Freddy no longer requires that, so we can properly express the service's required/optional properties. Unfortunately, the Swagger documentation is not always correct, so it falls on us (and our tests) to guess whether each property is required or optional.